src/app/profile/_components/ProfileInfo/ProfileInfo.tsx

```diff
@@ -0,0 +1,97 @@
+import { UserData } from '@/services/userService'
+import Button from '@/components/ui/Button'
+
+type UserProfileInfoProps = {
+  userData: UserData | null
+  formatDate: (dateString: string | null) => string
+  onEditClick: () => void
+  isLoading: boolean
+}
+
+const ProfileInfo = ({
+  userData,
+  formatDate,
+  onEditClick,
+  isLoading,
+}: UserProfileInfoProps) => {
+  return (
+    <>
+      {isLoading && <div className="text-center">Loading...</div>}
+      {userData && (
+        <div>
+          <h2 className="mb-8 text-2xl font-medium text-primary">
+            Personal details
+          </h2>
+          <ul className="mb-10 flex flex-col gap-y-6">
+            <li className="flex">
+              <span className="w-[152px] text-primary opacity-60">
+                First name:
+              </span>
+              {userData.firstName}
+            </li>
```

Comment:
This should be a separate component to avoid code duplication

Reply:
?

Reply:
fix